example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
```

Comment:
Although this is currently the `example` project, a lot of this code is what I picture getting moved into `useGravity`.